src/main/java/Task.java

```diff
@@ -0,0 +1,34 @@
+import java.io.Serializable;
+
+public class Task implements Serializable {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public void markAsDone() {
+        isDone = true;
+    }
+
+    public void markAsUndone() {
+        isDone = false;
+    }
+
+    @Override
+    public String toString() {
+        if (isDone) {
+            return "[X] " + description;
+        }
+        else {
```

Comment:
Similar to above, formatting of conditionals should be
```
if (condition) {
    statements;
} else {
    statements;
}
```

Reply:
Great spot! Thanks!